### PR TITLE
table styles

### PR DESCRIPTION
**Pull request by OksanaSidorova**

### CODE REVIEW

File: src/main/resources/static/images/icons/action_delete_axion.svg
Comment (arufanov) [inline src/main/resources/static/images/icons/action_delete_axion.svg:1]: @OksanaSidorova 
* should we move actions from images/icons  to images/actions folder? I suppose yes.

File: src/main/resources/static/css/application.css [hunk 1]
```diff
@@ -324,6 +324,100 @@ a.sidebar-link:hover {
...
+  background: #C65D0C;
+  color: #FFFFFF;
+}
+
+.paging-number-wrapper {
+  display: inline-block;
+  padding-right: 0.5em;
+}
+
+.paging-number-wrapper a {
```
Comment (arufanov) [inline src/main/resources/static/css/application.css:346]: @OksanaSidorova 
* should we move paging & list/table to independent css block similar to " *   Filter styles"

**Review by arufanov — CHANGES_REQUESTED** [review]

File: src/main/resources/static/images/icons/action_delete_axion.svg
Comment (OksanaSidorova) [reply src/main/resources/static/images/icons/action_delete_axion.svg:1]: @arufanov ok

File: src/main/resources/static/css/application.css (hunk 1 above)
Comment (OksanaSidorova) [reply src/main/resources/static/css/application.css:346]: @arufanov Ok, I'll move

**Review by arufanov — APPROVED** [review]
@OksanaSidorova 
* thanks!

**Pull request merged**